src/main/java/com/hbm/tileentity/turret/TileEntityTurretBase.java

```diff
@@ -100,10 +101,10 @@ public void updateEntity() {
 	}
 	
 	private boolean isInSight(Entity e) {
-		if(!(e instanceof EntityLivingBase) && !(e instanceof EntityMissileBaseAdvanced) && !(e instanceof EntityBomber) && !(e instanceof EntityMissileCustom))
+		if(!(e instanceof EntityLivingBase) && !(e instanceof EntityMissileBaseAdvanced) && !(e instanceof EntityBomber) && !(e instanceof EntityMissileCustom)&& !(e instanceof EntityMIRV))
 			return false;
 		
-		if(this instanceof TileEntityTurretCIWS && !(e instanceof EntityMissileBaseAdvanced) && !(e instanceof EntityBomber) && !(e instanceof EntityMissileCustom))
+		if(this instanceof TileEntityTurretCIWS && !(e instanceof EntityMissileBaseAdvanced) && !(e instanceof EntityBomber) && !(e instanceof EntityMissileCustom)&& !(e instanceof EntityMIRV))
```

Comment:
i don't even think this class is even used properly anymore

Reply:
Weirdly changing that one worked to make it target

Reply:
oh nvm thats for the phalanx